<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newObjectMaster_1</class>
 <widget class="QWidget" name="newObjectMaster_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>240</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Тип объекта</string>
   </property>
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Расширенное описание</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Имя объекта</string>
   </property>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Не более 30 символов</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Адрес в сети RS485</string>
   </property>
   <widget class="QSpinBox" name="addressSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>От 0 до 255</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
